src/github.com/getlantern/flashlight/util/common.go

```diff
@@ -0,0 +1,61 @@
+package util
+
+import (
+	"errors"
+	"github.com/getlantern/golog"
+	"github.com/mitchellh/go-homedir"
```

Comment:
We actually intentionally cross compile with cgo support because it improves things like DNS lookup performance.  That's why we use gonative (it has support for cross-compiling with cgo dependencies).